File: CILantroTools/cilantro-tools-web-ui/src/api/models/tests/input-output/elements/StringElement.ts
interface StringElement {
	type: 'String' | 'Char';
	name: string;
	minLength: number;
	maxLength: number;
	hasBigLetters: boolean;
	hasSmallLetters: boolean;
	hasDigits: boolean;
	hasSymbols: boolean;
}

export default StringElement;

File: CILantroTools/CILantroToolsWebAPI/CILantroToolsWebAPI/Models/Tests/InputOutput/Elements/StringElement.cs
namespace CILantroToolsWebAPI.Models.Tests.InputOutput.Elements
{
    public class StringElement : AbstractInputOutputElement
    {
        public string Name { get; set; }

        public int MinLength { get; set; }

        public int MaxLength { get; set; }

        public bool HasBigLetters { get; set; }

        public bool HasSmallLetters { get; set; }

        public bool HasDigits { get; set; }
    }
}